internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -605,7 +605,7 @@ func (opts *Opts) defaultRegions() ([]string, error) {
 // program if it receives an interrupt from the OS. We then handle this by printing
 // the dbUsername and dbPassword
 func (opts *Opts) setupCloseHandler() {
-	c := make(chan os.Signal)
+	c := make(chan os.Signal, 1)
```

Comment:
https://medium.com/orijtech-developers/sigchanyzer-static-analyzer-to-catch-loss-of-signals-from-use-of-unbuffered-signal-channel-to-e47529ca40a